src/components/BlogPost/BlogPostDialogMobile.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC } from 'react';
+
+import { Tag } from '../Tag/Tag';
+import { BlogPostDialogProps } from './BlogPostDialog';
+import { IconButton } from '../IconButton/IconButton';
+
+import { X, ChevronLeft, ChevronRight } from 'react-feather';
+import { Box, Dialog, makeStyles, Typography } from '@material-ui/core/';
+
+const useStyles = makeStyles((theme) => ({
+  buttons: {
+    backgroundColor: theme.palette.background.paper,
+    padding: theme.spacing(1, 2),
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  img: {
+    marginTop: theme.spacing(1, 2),
+    height: 180,
```

Comment:
If we set height: 180, we need to set width: 'auto' 
But maybe better idea is to set width: "100%" and height: "auto"